db/migrate/20220719131059_create_cohorts.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCohorts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :cohorts do |t|
+      t.belongs_to :track, null: false, foreign_key: true
+
+      t.string :slug, null: false, index: { unique: true }
+      t.string :name, null: false
+      t.column :capacity, :int, null: false, default: 0
+      t.datetime :begins_at, null: false
+      t.datetime :ends_at, null: false
```

Comment:
We don't yet do anything with these fields. I assume we want to show a special page if a cohort has not been opened or has ended?